Reordered sections within user docs (#1455)
conda-forge/conda-forge.github.io: src/user/introduction.rst

  Please refer to :ref:`multiple_channels` for pitfalls and more information.

> Can I contribute packages to conda-forge?

Reviewer: ```suggestion
Becoming Involved: How to contribute packages to conda-forge?
```

-----------------------------------------

> Yes, anyone can contribute packages to the ``conda-forge`` channel. 

Reviewer: ```suggestion
Anyone can contribute packages to the ``conda-forge`` channel. 
```

You don't have to be the upstream maintainer of a package in order to contribute it to ``conda-forge``. 

> To learn how you can contribute your first package read `the staging process <https://conda-forge.org/docs/maintainer/adding_pkgs.html#the-staging-process>`_.

Reviewer: ```suggestion
If you are interested in submitting a package to conda-forge, read `the staging process <https://conda-forge.org/docs/maintainer/adding_pkgs.html#the-staging-process>`_. for a better description of the process.
```
Maybe?
